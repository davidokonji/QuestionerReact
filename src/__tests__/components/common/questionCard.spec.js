import { shallow } from 'enzyme';
import React from 'react';
import Questions from '../../../components/common/questionCard';

describe('Question card component', () => {
  it('should render Question card correctly', () => {
    const wrapper = shallow(<Questions />);

    expect(wrapper).toMatchSnapshot();
  });
});
