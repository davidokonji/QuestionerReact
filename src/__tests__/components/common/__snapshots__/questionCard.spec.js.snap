// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Question card component should render Question card correctly 1`] = `
<div
  className="py-3"
>
  <div
    className="card col-12 col-md-10 shadow-sm"
  >
    <div
      className="card-body p-0 row"
    >
      <div
        className="col-1 col-md-1 d-flex flex-column justify-content-center my-2"
      >
        <p
          className="col"
        >
          <img
            alt="uparrow"
            className="h-25 arrow-icon"
            src={
              Object {
                "canInstrument": true,
                "createTransformer": [Function],
                "getCacheKey": [Function],
                "process": [Function],
              }
            }
          />
        </p>
        <p
          className="col mx-3"
        >
          0
        </p>
        <p
          className="col"
        >
          <img
            alt="downarrow"
            className="h-25 arrow-icon"
            src={
              Object {
                "canInstrument": true,
                "createTransformer": [Function],
                "getCacheKey": [Function],
                "process": [Function],
              }
            }
          />
        </p>
      </div>
      <div
        className="col-9 col-md-9 row"
      >
        question
      </div>
    </div>
  </div>
</div>
`;
